studio/schemas/documents/sections/readyToInvest.js

```diff
@@ -0,0 +1,41 @@
+import { SplitHorizontalIcon } from '@sanity/icons'
+import supportedLanguages from '../../supportedLanguages';
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault);
+
+export default {
+  type: 'document',
+  name: 'readyToInvest',
+  title: 'Ready To Invest',
+  icon: SplitHorizontalIcon,
+  fields: [
+    {
+      name: 'heading',
+      type: 'localePortableText',
+      title: 'Heading (*)',
+      validation: Rule => Rule.error('Information required.').required(),
+    },
+    {
+      name: 'buttonText',
+      type: 'localePortableText',
+      title: 'Button text'
+    },
+    {
+      name: 'backgroundColor',
+      type: 'string',
+      title: 'Background Color',
+      description: "Use this field to input the background color in the same pattern as '#FFFFFF'.",
```

Comment:
Add the information about the default color, similarly as I've updated here at `studio\schemas\documents\sections\textSeparator.js`
![image](https://user-images.githubusercontent.com/104107061/193573492-2404abf4-8026-41b5-8d08-26bab9c3e1cb.png)